tex/set-theory/CH.tex

```diff
@@ -73,7 +73,7 @@ \section{Adding in reals}
 but with $\omega_2$ many real numbers instead of just one.
 In light of this, consider in $M$ the poset
 \[
-	\Po = \opname{Add} \left( \omega_2, \omega \right)
+	\Po = \opname{Add}(\omega, \omega_2)
```

Comment:
Swapping the order (Page 670 of Jech defines Add(α, β) as adding β subsets of α. In the Napkin, Add(α, β) in general was never defined.)

Changing \left( to ( is just to remove the unnecessary space before the (.